paybybank/src/main/java/com/adyen/checkout/paybybank/internal/ui/view/PayByBankView.kt

```diff
@@ -6,7 +6,7 @@
  * Created by ozgur on 28/9/2022.
  */
 
-package com.adyen.checkout.paybybank
+package com.adyen.checkout.paybybank.internal.ui.view
```

Comment:
Can this class be internal?

Reply:
I am not sure. Are there any cases in which we directly use the view instead of Adyen component view?

Reply:
I don't think so